packages/target-graph/tests/TargetGraphBuilder.test.ts

```diff
@@ -33,11 +34,11 @@ describe("target graph builder", () => {
 
     const targetGraph = builder.buildTargetGraph(["build"]);
 
-    expect(targetGraph.targets.size).toBe(2);
+    expect(targetGraph.targets.size).toBe(3);
 
     expect(targetGraph.dependencies.length).toBe(3);
-    expect(targetGraph.dependencies).toContainEqual([START_TARGET_ID, "a#build"]);
-    expect(targetGraph.dependencies).toContainEqual([START_TARGET_ID, "b#build"]);
+    expect(targetGraph.dependencies).toContainEqual([getStartTargetId(), "a#build"]);
```

Comment:
minor comment but might be nice to take an inlineSnapshot here to more clearly explain the validation.

Why is targets increasing from 2 to 3?

What does getStartTargetId return? (maybe you should inline the expectation rather than deferring expectation to a utility.)

Reply:
(if you're worried about paths sneaking into the inlineSnapshot, you could filter those as well, something like `removePathsFromOutput(json, [ 'key/matches/containing/paths' ])`